examples/abitest/example.toml

```diff
@@ -1,5 +1,9 @@
 name = "abitest"
 
+[application]
+manifest = "examples/abitest/config/config.toml"
+out = "examples/abitest/bin/config.bin"
+
 [modules]
 module_0 = { Cargo = { cargo_manifest = "examples/abitest/module_0/rust/Cargo.toml" } }
 module_1 = { Cargo = { cargo_manifest = "examples/abitest/module_1/rust/Cargo.toml" } }
```

Comment:
pre-existing nit below: `[clients] cpp` is not accurate, it would be nice to sync this with #1170.

Reply:
I think this is accurate, i.e. it is the `cpp` client, and it is built via `bazel`, which is what the details of the entry says.

Reply:
Ah, my bad, this isn't a fixed schema &ndash; this comment belongs elsewhere&hellip;